src/atoms/Checkbox/Input.web.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import Backdrop from './Backdrop';
+
+const Input = styled.input.attrs({
+  type: 'checkbox',
+})`
+  position: absolute;
+  width: 0px;
+  height: 0px;
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
```

Comment:
Why is this needed?

Reply:
This was added for clip.
> Note: The clip property does not work if "overflow:visible".
https://www.w3schools.com/cssref/pr_pos_clip.asp

clip-path is not supported in IE. clip is supported although it is deprecated. should we remove the usage of clip or keep it for safety?